### PR TITLE
feat: support seeding external database

### PR DESCRIPTION
**Pull request by mook-as**
## Description
This adds the option to seed the external database.

## Motivation and Context
Fixes most of #629 (needs docs before closing).

## How Has This Been Tested?
On minikube:
- Used `scripts/deploy-mysql.sh` (but exiting before seeding) to deploy a MySQL
- Deployed KubeCF using the external database
- Smoke tests.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code has security implications.
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
Will make doc PR after this gets merged; hence intentionally not marking this PR as closing #629.

### CODE REVIEW

**Review by jandubois — COMMENTED** [review]
Just publishing what I have so far; I'm still at `5 / 12 files reviewed`.

**Review by mook-as — COMMENTED** [review]
Changes made; rebased on top of `master` to resolve merge conflicts.

File: chart/templates/database/db-statefulset.yaml [hunk 1]
```diff
@@ -48,22 +38,12 @@ spec:
       replicas: {{ $replicas }}
       selector:
         matchLabels:
-          app: database
-          app.kubernetes.io/instance: {{ .Release.Name | quote }}
-          app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
-          app.kubernetes.io/name: {{ include "kubecf.fullname" . }}
-          app.kubernetes.io/version: {{ default .Chart.Version .Chart.AppVersion | quote }}
-          helm.sh/chart: {{ include "kubecf.chart" . }}
+          {{- list . "database" | include "component.labels" | nindent 10 }}
```
Comment (mook-as) [inline chart/templates/database/db-statefulset.yaml:41]: This should be okay as the old selector included the version, which would not have worked across upgrades; therefore modifying the statefulset selector must be okay.

File: chart/templates/database/_database.tpl [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{{- /*
+==========================================================================================
+| _database.load $
```
[the remark this answers is not in the document]
Comment (jandubois) [reply chart/templates/database/_database.tpl]: > have not attempted the larger refactoring

Didn't expect you to anyways.

**Review by jandubois — APPROVED** [review]

**Pull request merged**